components/PaywallModal.vue

```diff
@@ -0,0 +1,308 @@
+<template>
+  <UModal
+    :full-screen="props.isFullScreen || isMobileScreen"
+    :dismissible="props.dismissible"
+    :ui="{ content: 'sm:rounded-2xl overflow-x-hidden sm:w-[90vw] max-w-[840px]' }"
```

Comment:
prefer using `phone:` `tablet:` `laptop:` for responsive selector

Reply:
[🎨 Refine UI style formatting](https://github.com/likecoin/3ook-com/pull/57/commits/ebc6d70152b905cd19f5c2c4e7feedb95f77cb1e)